store/tikv/region_request.go

```diff
@@ -195,11 +195,9 @@ func (s *RegionRequestSender) onRegionError(bo *Backoffer, ctx *RPCContext, regi
 		// Retry if error is `NotLeader`.
 		log.Debugf("tikv reports `NotLeader`: %s, ctx: %v, retry later", notLeader, ctx)
 		s.regionCache.UpdateLeader(ctx.Region, notLeader.GetLeader().GetStoreId())
-		if notLeader.GetLeader() == nil {
-			err = bo.Backoff(BoRegionMiss, errors.Errorf("not leader: %v, ctx: %v", notLeader, ctx))
-			if err != nil {
-				return false, errors.Trace(err)
-			}
+
+		if err = bo.Backoff(BoRegionMiss, errors.Errorf("not leader: %v, ctx: %v", notLeader, ctx)); err != nil {
```

Comment:
maxSleep of BoRegionMiss is 500ms, is that enough?

Reply:
@winkyao 
I think this `maxSleep` value is determined by the type of `bo`. `BoRegionMiss` affects the `cap` of every sleep.